config/action.d/badips.conf

```diff
@@ -10,7 +10,7 @@
 
 [Definition]
 
-actionban = curl --fail  --user-agent "fail2ban v0.8.12" http://www.badips.com/add/<category>/<ip>
+actionban = curl --fail  --user-agent "<agent>" http://www.badips.com/add/<category>/<ip>
```

Comment:
Recent changes below reminded me that for any if those action parameters we are providing defaults in the same file.